src/java.desktop/share/classes/java/awt/TextField.java

```diff
@@ -488,6 +489,15 @@ public Dimension minimumSize() {
         }
     }
 
+    @Override
+    public synchronized void setEditable(boolean b) {
+        super.setEditable(b);
+        Color defaultBackground = UIManager.getColor("TextField.background");
```

Comment:
It still uses UIManager.getColor here and this property is again set in L&Fs mentioned for`TextArea.background`

Reply:
UIManager removed from textfield as well now